src/compiler/symb-define.bas

```diff
@@ -124,11 +122,11 @@ private function hDefErr_cb ( ) as string
 	function = str( res )
 end function
 
-private function hDefLang_cb ( ) as string
+private function hDefLang_cb() as string static
```

Comment:
Is `static` really useful here? As far as I remember, it only affects normal variables, not temps or function results.